ldes-server-infra-mongo/mongo-ingest-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/mapper/MemberEntityMapper.java

```diff
@@ -0,0 +1,28 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.ingest.mapper;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.ingest.entities.Member;
+import be.vlaanderen.informatievlaanderen.ldes.server.ingest.entities.MemberEntity;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFDataMgr;
+import org.apache.jena.riot.RDFParserBuilder;
+import org.springframework.stereotype.Component;
+
+import java.io.StringWriter;
+
+@Component
+public class MemberEntityMapper {
```

Comment:
Would personally avoid the Component annotation here (is easier if later on somebody creates an integration test for MemberRepositoryImpl). And just create an instance where this is used.

Reply:
To be discussed. I don't agree with this statement. Dependency injection would make testing easier and not harder. It also removes prevents the classes using this mapper (currently only MemberRepositoryImpl) not to require any information on how the mapper works, what it needs or how it needs to be created. If/when we need to add a dependency to the mapper (for example to map a new field), we would have to refactor everywhere the mapper is used and also figure out how to get that dependency in there. The MemberRepositoryImpl would become tightly coupled with the mapper. It is possible we may want to use this mapper on other locations as well, augmenting the problem. All the advantages of using DI apply here.